VimeoNetworking/Sources/Constants.swift

```diff
@@ -27,7 +27,7 @@
 import Foundation
 
  /// Base URL for the Vimeo API
-let VimeoBaseURL = URL(string: "https://api.vimeo.com")!
+public let VimeoBaseURL = URL(string: "https://api.vimeo.com")!
```

Comment:
Why do we need to change the access level here?

Reply:
@nicolelehrer this change is only temporary. I'm aware that Jason is changing `Vimeo-iOS` and some submodules so that we don't use this public URL anymore. Once those changes are merged, I will open an PR to revert this change.

Reply:
Add a TODO so it doesn't get lost?